app/assets/javascripts/controllers/navigation/active_item_controller.js

```diff
@@ -0,0 +1,36 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class ActiveItemController extends Controller {
+  static targets = ["menu"];
+  static values = {
+    cssClass: { type: String, default: "active" },
+    addToParents: Boolean
+  }
+
+  connect() {
+    console.error("connect")
+    const item = this.getParent(this.activeLink)
+    this.setActiveItem(item);
+  }
+
+  get activeLink() {
+    return this.menuTarget.querySelector("a[href='" + location.pathname + "']")
+  }
+
+  getParent(item) {
+    return item.closest("li")
```

Comment:
I don't think you've run the linter, there are inconsistently semicolons on line ends.